website/docs/developers/governance.mdx

```diff
@@ -56,14 +64,40 @@ The helper `build_commitments_from_votes` can be called in simulation to reprodu
 
 ## Weights & quorum
 
-The effective weight of a vote is given by [`get_max_weight`](./membership.mdx). The contract does **not** impose a fixed quorum rule – instead it considers the weighted tallies:
+The effective weight of a vote is given by [`get_max_weight`](./membership.mdx). The contract implements a **supermajority governance model** that requires broad consensus before approving any proposal.
+
+### Voting Decision Logic
+
+The contract considers the weighted tallies using supermajority rules:
 
 - `approve > (abstain + reject)` → **Approved**
 - `reject > (abstain + approve)` → **Rejected**
 - Otherwise → **Cancelled** (including when approve = reject)
 
+This means that to approve a proposal, the approve votes must exceed the combined weight of all reject and abstain votes. Similarly, to reject a proposal, the reject votes must exceed the combined weight of all approve and abstain votes.
+
+### Why Supermajority?
+
+This supermajority approach ensures that:
+
+- **Broad consensus** is required before making changes
+- **Abstain votes matter** - they represent active participation but uncertainty
+- **Conservative governance** - proposals need clear support to pass
+- **Protection against hasty decisions** - changes require overwhelming support
+
 Custom quorum logic (percentage, thresholds…) can be built off-chain before deciding whether to call `execute`.
 
+### Voting Eligibility
+
+To cast a vote on a proposal, users must:
+
+- Be registered as a member using `add_member`
+- Have one or more badges assigned for the project (see [Membership & Badges](./membership.mdx))
```

Comment:
People can have no badges, they just get weight as 1. Maybe I should revisit the values. I will think about that for some follow ups.